code/src/App.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react'
-import data from './data.json'
 
-console.log(data)
+import { AlbumWrapper } from 'components/AlbumWrapper'
+import { Sidebar } from 'components/Sidebar'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className='album-wrapper'>
+      <img
+        className='menu'
+        src='/icons/dots.svg'
+        alt='dot icon'
+        onClick={e =>
+          (e.target.className = e.target.className.includes('open') ? 'menu' : 'menu open')
+        }
+      />
+      <div className='container'>
+        <h1 className='title'>New albums</h1>
+        <div className='album-grid'>
+          <AlbumWrapper type={'album'} />
```

Comment:
Nice way of solving this! To reuse the AlbumWrapper and specify album type separately.